src/axom/inlet/examples/user_defined_type.cpp

```diff
@@ -184,12 +184,12 @@ struct ThermalSolver
   static void defineSchema(inlet::Table& schema)
   {
     // _inlet_userdef_simple_usage_start
-    auto& mesh_table = schema.addTable("mesh", "Information about the mesh");
+    auto& mesh_table = schema.addStruct("mesh", "Information about the mesh");
```

Comment:
Calling this `mesh_table` is a tad confusing here due to them not lining up now.

Reply:
I think keeping the `Table` class name is fine; the intent of the change is to separate the addition of a `Table` (which is specific to the internals of the Inlet hierarchy) and the addition of a struct/record type to the input schema.

I agree that this should probably be renamed though, maybe `mesh_schema`?

Reply:
`mesh_schema` makes me feel better. which is what is important.